### PR TITLE
More changes

### PR DESCRIPTION
**Pull request by 0xprinc**

### CODE REVIEW

File: src/dex/fluid-dex/fluid-dex.ts [hunk 1]
```diff
@@ -119,9 +119,9 @@ export class FluidDex extends SimpleExchange implements IDex<FluidDexData> {
   // for pricing requests. It is optional for a DEX to
   // implement this function
   async initializePricing(blockNumber: number) {
-    Object.entries(this.eventPools).forEach(([id, eventPool]) => {
-      eventPool.getStateOrGenerate(blockNumber, false);
-    });
+    // Object.entries(this.eventPools).forEach(([id, eventPool]) => {
```
Comment (bergben) [inline src/dex/fluid-dex/fluid-dex.ts]: commented out

File: src/dex/fluid-dex/fluid-dex.ts [hunk 2]
```diff
@@ -249,7 +249,7 @@ export class FluidDex extends SimpleExchange implements IDex<FluidDexData> {
 
       return [
         {
-          prices: prices.filter((price): price is bigint => price !== null), // to be done
+          prices: prices,
           unit: getBigIntPow(
             (side === SwapSide.SELL ? destToken : srcToken).decimals,
           ), // to be done
```
Comment (bergben) [inline src/dex/fluid-dex/fluid-dex.ts]: to be done

File: src/dex/fluid-dex/fluid-dex-events.test.ts [hunk 3]
```diff
@@ -136,6 +136,90 @@ describe('FluidDex EventPool Mainnet', function () {
           ([eventName, blockNumbers]: [string, number[]]) => {
             describe(`${eventName}`, () => {
               blockNumbers.forEach((blockNumber: number) => {
+                // it(`State after ${blockNumber}`, async function () {
```
Comment (bergben) [inline src/dex/fluid-dex/fluid-dex-events.test.ts]: why is there so much commented out code again

File: src/dex/fluid-dex/fluid-dex-pool.ts [hunk 4]
```diff
@@ -50,6 +47,15 @@ export class FluidDexEventPool extends StatefulEventSubscriber<FluidDexPoolState
     this.handlers['LogOperate'] = this.handleOperate.bind(this);
   }
 
+  private padHexTo32Bytes(hex: string): string {
```
Comment (bergben) [inline src/dex/fluid-dex/fluid-dex-pool.ts]: such things should not be necessary. also decoding usually works via ABI and interface. Feel like there should be better solutions than manually parsing from raw

https://docs.ethers.org/v5/api/utils/bytes/#utils-hexZeroPad

File: src/dex/fluid-dex/fluid-dex-events.test.ts (hunk 3 above)
Comment (0xprinc) [reply src/dex/fluid-dex/fluid-dex-events.test.ts]: changed!!

File: src/dex/fluid-dex/fluid-dex-pool.ts (hunk 4 above)
Comment (0xprinc) [reply src/dex/fluid-dex/fluid-dex-pool.ts]: removed this function

**Pull request merged**